Matrix/src/Matrix.cpp

```diff
@@ -105,6 +105,23 @@ Matrix &Matrix::operator+(Matrix &mat_1)
 	return mat_1;
 }
 
+Matrix &Matrix::operator+=(const Matrix& RHS)
+{
+	size_t nRows = numberOfRows();
+	size_t nColumns = numberOfCols();
+
+	for (int i = 0; i < nRows; ++i)
+	{
+		for (int j = 0; j < nColumns; ++j)
+		{
+			(*this)(i, j) += RHS(i, j);
```

Comment:
The operators has right to reach to the private items. Therefore you can directly update the main matrix:
`m_matrix[i][j] += RHS(i, j); ` (@JusticeBoi )

Reply:
I would guess they end up producing the same assembly, so it is a matter of style. Perhaps @sariug's suggestion looks a bit cleaner.